src/main/java/com/revature/parasol/controllers/LoginController.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * 
+ */
```

Comment:
Controller for logging in. This should do all of the spring setup and log the user in.